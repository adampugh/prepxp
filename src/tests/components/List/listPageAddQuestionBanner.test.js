import React from "react";
import { shallow } from "enzyme";

import ListPageAddQuestionBanner from "../../../components/List/listPageAddQuestionBanner";

describe("<ListPageAddQuestionBanner />", () => {
    it("should match snapshot", () => {
        const wrapper = shallow(<ListPageAddQuestionBanner />);
        expect(wrapper).toMatchSnapshot();
    });
});