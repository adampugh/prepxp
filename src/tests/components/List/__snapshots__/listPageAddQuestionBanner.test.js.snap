// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListPageAddQuestionBanner /> should match snapshot 1`] = `
<div
  className="listPage__addQuestion"
>
  <div
    className="container-s"
  >
    <img
      alt="people being interviewed"
      src="interview2.png"
    />
    <h1>
      Add a question
    </h1>
    <p>
      Add a question below and add it to your list
    </p>
    <div
      className="listPage__addQuestion__input"
    >
      <div>
        <input
          id="questionInput"
          onChange={[Function]}
          onKeyDown={[Function]}
          placeholder="What weaknesses do you have?"
          type="text"
        />
        <button
          className="btn"
          onClick={[Function]}
        >
          Add Question
        </button>
      </div>
    </div>
  </div>
  <div
    className="listPage__addQuestion__whiteElipse"
  />
</div>
`;
